api/CSSMatrix.json

```diff
@@ -45,6 +45,9 @@
           },
           "webview_android": {
             "version_added": null
+          },
+          "samsunginternet_android": {
```

Comment:
Lets try to avoid null entries, if we can discover if we support it that would be great.